### ISSUE TITLE
How to lint golangci

### ISSUE THREAD

**Issue by ray-x** [mfussenegger/nvim-lint#27]
I think the command to run is 
lua require'lint'.try_lint('golangcilint')
But there is nothing come out after the command. Is there anything else need to do other than:

```
  require('lint').linters_by_ft = {
    markdown = {'vale',},
    go = {'golangcilint'},
  }
```

**Comment by mfussenegger** [comment]
That should be enough. Are you sure there are warnings or errors that should be reported in your case? If there is nothing to report, you won't see anything happening.

**Comment by ray-x** [comment]
Confirm. it caused by incorrect setup of golangci-lint.
